geoips/geoips_utils.py

```diff
@@ -391,3 +394,105 @@ def merge_nested_dicts(dest, src, in_place=True, replace=False):
         raise
     if not in_place:
         return final_dest
+
+
+def expose_geoips_commands(pkg_name=None, test_log=None):
+    """Expose a list of commands that operate in the GeoIPS environment.
+
+    Where, these commands are defined under 'pyproject.toml:[tool.poetry.scripts]',
+    or 'pyproject.toml:[project.entry-points.console_scripts]'
+
+    Parameters
+    ----------
+    pkg_name: str (default = None)
+        - The name of the GeoIPS Plugin Package whose command's will be exposed.
+        - If None, assume this was called via the commandline and retrieve package_name
+          via that manner. Otherwise use the supplied package_name.
+    test_log: logging.Logger (default = None)
+        - If provided, use this logger instead. This is added as an optional argument
+          so we can check the output of this command for our Unit Tests.
```

Comment:
Is there a use case for this argument other than for testing? If not, maybe it would be better to either make the argument `_test_log` to indicate that it shouldn't really be used by users/devs or just collected from `**kwargs`.

Reply:
I can make it ``_test_log`` if you prefer. Just let me know and those changes will be made if you want them.

Reply:
Yes please!

Reply:
Changed that to a private variable. I also fixed the remaining unit test. Just replaced relative imports with absolute imports from modules which made use of them.